unison-runtime/src/Unison/Runtime/Machine.hs

```diff
@@ -129,19 +156,23 @@ baseCCache sandboxed = do
     <*> newTVarIO builtinTypeNumbering
     <*> newTVarIO baseSandboxInfo
   where
+    cacheableCombs = mempty
     ffuncs | sandboxed = sandboxedForeigns | otherwise = builtinForeigns
     noTrace _ _ = NoTrace
     ftm = 1 + maximum builtinTermNumbering
     fty = 1 + maximum builtinTypeNumbering
 
     rns = emptyRNs {dnum = refLookup "ty" builtinTypeNumbering}
 
-    combs :: EnumMap Word64 RCombs
-    combs =
-      ( mapWithKey
+    srcCombs :: EnumMap Word64 Combs
```

Comment:
I now include all the original non-recursive Combs in the CCache now because it makes it much easier to serialize